spellchecker/symspell.py

```diff
@@ -62,24 +63,28 @@ def __init__(self, initial_capacity=16, max_dictionary_edit_distance=2,
         self._distance_algorithm = DistanceAlgorithm.DAMERUAUOSA
         self._max_length = 0
 
-    def create_dictionary_entry(self, key, count):
-        """Create/Update an entry in the dictionary.
-        For every word there are deletes with an edit distance of
-        1..max_edit_distance created and added to the dictionary. Every delete
+    def create_dictionary_entry(self, key: str, count: int, canonical_term: str=None) -> bool:
+        """
+        Create/Update an entry in the dictionary.
+        For every word there are deletes with an edit distance of 1..max_edit_distance
+        created and added to the dictionary. Every delete
         entry has a suggestions list, which points to the original term(s) it
         was created from. The dictionary may be dynamically updated (word
         frequency and new words) at any time by calling
         create_dictionary_entry
 
-        Keyword arguments:
-        key -- The word to add to dictionary.
-        count -- The frequency count for word.
+       Args:
+           key: The word to add to dictionary.
+           count: The frequency count for word.
+           canonical_term: This is an optional term (or multi-terms) that will used as suggestion instead of
+        the original word (e.g. Saint-Laurent instead of St-Laurent). By default, the key value is used
```

Comment:
Soft comments:

if the description of a param requires more than one line indent the following lines (see: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

Also the text of your Returns is not indented by 4.